suite-common/message-system/src/messageSystemThunks.ts

```diff
@@ -19,6 +18,34 @@ import {
     selectMessageSystemCurrentSequence,
 } from './messageSystemSelectors';
 
+const getConfigJws = async () => {
+    try {
+        const response = await scheduleAction(signal => fetch(CONFIG_URL_REMOTE, { signal }), {
+            timeout: FETCH_TIMEOUT,
+        });
+
+        if (!response.ok) {
+            throw Error(response.statusText);
+        }
+
+        const configJws = await response.text();
+
+        return {
+            configJws,
+            isRemote: true,
+        };
+    } catch (error) {
+        console.error(`Fetching of remote JWS config failed: ${error}`);
+
+        const { jws } = await import('../files/config.v1');
```

Comment:
Not sure about this, but I think this won't work in mobile (lazy imports not supported yet, but I think they could be supported in next version of RN)

Reply:
I wasn't sure too, but I tried it and it worked - I got the content of the local config to state in the mobile app. But I've only tried the local dev build. Do you think it might stop working in the "production" build?

Reply:
Sorry my bad it's actually static import so it will work. Dynamic import have variables in path.